projects/arlas-toolkit/src/lib/components/config-manager/config-menu/config-menu.component.ts

```diff
@@ -62,7 +64,7 @@ export class ConfigMenuComponent implements OnInit {
         if (action.url && action.config && action.config.id && action.configIdParam) {
 
           let url = action.url.concat('/?' + action.configIdParam + '=').concat(action.config.id);
-          if (!!action.config.org) {
+          if (!!action.config.org && action.config.org !== 'no_organisation') {
```

Comment:
It's the only place where `no_organisation` is used ?
If not, maybe it could be nice to create a const

Reply:
resolved